parser-worker/createDetailedGeometry.ts

```diff
@@ -91,10 +91,10 @@ export async function createDetailedGeometry(
         for (let i = 0; i < fileMeshIndices.length; i++) {
           const meshIdx = meshIndices[fileMeshIndices[i]];
           const treeIndex = treeIndices[meshIdx];
-          const instanceMatrix = instanceMatrices.slice(meshIdx * 16, meshIdx * 16 + 16);
+          const instanceMatrix = instanceMatrices.subarray(meshIdx * 16, meshIdx * 16 + 16);
```

Comment:
This is an optimization, subarray() is a lot faster than slice():
https://jsbench.me/56knoajd9a/1